ui/components/multichain/pages/send/components/domain-input-resolution-cell.tsx

```diff
@@ -120,24 +124,14 @@ export const DomainInputResolutionCell = ({
                 size={AvatarIconSize.Xs}
                 className="multichain-send-page__recipient__item__avatar"
                 backgroundColor={BackgroundColor.infoDefault}
+                color={IconColor.infoInverse}
                 borderColor={BorderColor.backgroundDefault}
                 borderWidth={2}
-                iconProps={{
-                  color: IconColor.infoInverse,
-                  style: { width: '12px', height: '12px' },
-                  name: IconName.Snaps,
-                }}
               />
             }
-            positionObj={{
-              bottom: '25%',
-              right: '10%',
-            }}
-            badgeContainerProps={{
-              className: 'multichain-send-page__recipient__item__badge',
-            }}
```

Comment:
We could use the default sizes here it would be good to understand why these custom sizes are required it may be that we need to update the sizes on DS side